macros/src/actix.rs

```diff
@@ -481,27 +481,32 @@ pub fn emit_v2_definition(input: TokenStream) -> TokenStream {
     let mut props_gen = quote! {};
 
     match &item_ast.data {
-        Data::Struct(ref s) => match &s.fields {
-            Fields::Named(ref f) => handle_field_struct(f, &props, &mut props_gen),
-            Fields::Unnamed(ref f) => {
-                if f.unnamed.len() == 1 {
-                    handle_unnamed_field_struct(f, &mut props_gen)
-                } else {
+        Data::Struct(ref s) => {
+            props_gen.extend(quote!(
+                schema.data_type = Some(DataType::Object);
+            ));
```

Comment:
The diff is so complex... but I actually just added the 3 likes above + fixed the test cases